www/.jshintrc

```diff
@@ -7,6 +7,7 @@
 	"nonew": true,
 	"undef": true,
 	"unused": true,
+	"esversion": 6,
```

Comment:
This seems like a bad idea unless all ES6 features are available in all iOS versions you support. By the byte, long term I hope we can switch to ESLint!